docs/index.md

```diff
@@ -2,33 +2,72 @@
 hide:
 - toc
 ---
-
 ![logo](imgs/logo.png){ align=right }
 
-Trivy (tri pronounced like trigger, vy pronounced like envy) is a comprehensive security scanner. It is reliable, fast, extremely easy to use, and it works wherever you need it.
+# Trivy Documentation
+
+👋 Welcome to Trivy Documentation! To help you get around, please notice the different sections at the top global menu:
+
+- You are currently in the [Getting Started] section where you can find general information and help with first steps.
+- In the [Tutorials] section you can find step-by-step guides that help you accomplish specific tasks.
+- In the [CLI] section you will find the complete reference documentation for all of the different features and settings that Trivy has to offer.
+- In the [Ecosystem] section you will find how Trivy works together with other tools and applications that you might already use.
+- In the [Contributing] section you will find instructions about developing Trivy, and contribution guidelines.
+
+# About Trivy
 
-Trivy has different scanners that look for different security issues, and different targets where it can find those issues.
+Trivy is a comprehensive and versatile security scanner. Trivy has *scanners* that look for security issues, and *targets* where it can find those issues.
 
-Targets:
+Targets (what Trivy can scan):
 
 - Container Image
 - Filesystem
 - Git repository (remote)
-- Kubernetes cluster or resource
```

Comment:
Maybe we want to point out both separately -- Trivy scans running Kubernetes workloads inside the cluster but also static Kubernetes manifests

Reply:
> workloads inside the cluster but also static Kubernetes manifests

by "static manifests" you mean k8s yaml on disk? in this case, it a fs target no? or you meant scanning a resource in k8s, in this case I didn't get the contrast with "kubernetes workloads" in your suggestion.

Reply:
I mean that the 'trivy config' command can check Kubernetes manifests and that the 'k8s' command can check running workloads